scripts/sql/normalize_sql.py

```
@@ -0,0 +1,379 @@
+    # 양옆 단어 경계에서만 치환 (식별자 토큰 전체에만 매칭)
+    s = re.sub(rf'\b{re.escape(bad)}\b', good, s)
+  return s
+
+
+def _coarse_cleanup(s: str) -> str:
+  s = _strip_comments(s)
+  s = s.replace('"', '')
+  s = s.replace('\r', '')
+  lines = []
+  for line in s.splitlines():
+    l = line.strip()
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)